src/main/java/maksym/fedorenko/bookstore/dto/book/BookDto.java

```diff
@@ -1,9 +1,17 @@
 package maksym.fedorenko.bookstore.dto.book;
 
 import java.math.BigDecimal;
+import java.util.List;
+import lombok.Data;
 
-public record BookDto(
-        Long id, String title, String author, String isbn,
-        BigDecimal price, String description, String coverImage
-) {
+@Data
+public class BookDto {
```

Comment:
Why not record?

Reply:
Because I had to add net mapper:
@AfterMapping default void setCategoryIds(@MappingTarget BookDto bookDto, Book book)
that sets category IDs and with record I can use setter, so I had to update record to general class or use a Constructor to create a new record. And the second option seemed more inconvenient to me